lib/Mojo/BaseUtil.pm

```diff
@@ -0,0 +1,71 @@
+package Mojo::BaseUtil;
+use Mojo::Base -strict;
+
+use Exporter  qw(import);
+use Sub::Util qw(set_subname);
+
+our @EXPORT_OK = (qw(monkey_patch));
+
+
+sub class_to_path { join '.', join('/', split(/::|'/, shift)), 'pm' }
+
+sub monkey_patch {
+  my ($class, %patch) = @_;
+  no strict 'refs';
+  no warnings 'redefine';
+  *{"${class}::$_"} = set_subname("${class}::$_", $patch{$_}) for keys %patch;
+}
+
+1;
+
+=encoding utf8
+
+=head1 NAME
+
+Mojo::BaseUtil - Common utility functions used in Mojo::Base, re-exported in Mojo::Util.
+
+=head1 SYNOPSIS
+
+  use Mojo::BaseUtil qw(class_to_patch monkey_path);
+
+  my $path = class_to_path 'Foo::Bar';
+  monkey_patch 'MyApp', foo => sub { say 'Foo!' };
+
+=head1 DESCRIPTION
+
+L<Mojo::BaseUtil> provides a C<class_to_path> and the C<monkey_patch> function for L<Mojo>.
```

Comment:
It may be worth detailing here the purpose of this module: to provide functions for both Mojo::Base and Mojo::Util, so Mojo::Base does not have to load the rest of Mojo::Util.

Reply:
That's a good idea. I more or less included your sentence literally as I think it's already phrased in the best understandable way.